common/data/series.js

```diff
@@ -0,0 +1,170 @@
+        url: '',
+        description: '',
+        datePublished: new Date('2017-06-29')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'The colonist who faced the blue terror',
+        // url: 'outsiders-the-colonist',
+        url: '',
+        description: '',
+        datePublished: new Date('2017-07-06')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'The child whose town rejected vaccines',
+        // url: 'outsiders-the-child',
+        url: '',
+        description: '',
+        datePublished: new Date('2017-07-13')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'The prostitute whose pox inspired feminists',
+        // url: 'outsiders-the-prostitute',
+        url: '',
+        description: '',
+        datePublished: new Date('2017-07-20')
+      })
+    ])
+  }),
+  ({
+    url: 'electric-sublime',
+    name: 'Electric Sublime',
+    commissionedLength: getSeriesCommissionedLength('electric-sublime'),
+    color: 'turquoise',
+    items: List([
+      ({
+        contentType: 'article',
+        headline: 'Thunderbolts and lightning',
+        // url: 'thunder-bolts-and-ligtning',
+        url: '',
+        description: '',
+        datePublished: new Date('2017-03-30')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'Eels and feels',
+        // url: 'eels-and-feels',
+        url: '',
+        description: '',
+        datePublished: new Date('6 April 2017')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'Charged bodies',
+        // url: 'charged-bodies',
+        url: '',
+        description: '',
+        datePublished: new Date('13 April 2017')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'The current that kills',
+        // url: 'the-current-that-kills',
+        url: '',
+        description: '',
+        datePublished: new Date('20 April 2017')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'Dazzling luxury',
+        // url: 'dazzling-luxury',
+        url: '',
+        description: '',
+        datePublished: new Date('27 April 2017')
+      }),
+      ({
+        contentType: 'article',
+        headline: 'Titans in the landscape',
+        // url: 'titans-in-the-landscape',
+        url: '',
+        description: '',
+        datePublished: new Date('4 May 2017')
+      })
+    ])
+  })
+]);
+
+export const collectorsPromo = {
+  modifiers: ['standalone'],
+  url: 'http://digitalstories.wellcomecollection.org/pathways/2-the-collectors/',
+  title: 'The Collectors',
+  description: 'Searchers, secrets and the power of curiosity.',
+  image: {
+    contentUrl: 'https://wellcomecollection.files.wordpress.com/2017/03/the-collectors-promo.jpg',
+    width: 1600,
+    height: 900
+  },
+  positionInSeries: 1,
+  series: [{
+    color: 'turquoise',
+    commissionedLength: 1,
+    items: {
+      size: 1
+    }
+  }]
+};
```

Comment:
Why have we moved this over?
If ew want to - let's clean it up as this has loads of stuff that should be deprecated.

How do we feel about forcing flow on these files too?